### PR TITLE
Fix list bullet

### PR DESCRIPTION
**Pull request by mdbenjam**
A recent css change to enable factoring out components to the caseflow frontend toolkit resulted in a visual regression of our category selectors in Reader. 

image
^ Ugly AF

screen shot 2018-01-29 at 3 12 18 pm
Now it's pretty again!

**Test Plan**
- [ ] Examine the document viewer category checkboxes and ensure it looks as it did before the regression.
- [ ] Examine the document list dropdown filter checkboxes for categories and tags and ensure it looks like it did before the regression.

### CODE REVIEW

File: client/app/reader/DropdownFilter.jsx [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+
+const dropdownFilterViewListStyle = css({
+  margin: 0
+});
+const dropdownFilterViewListItemStyle = css(
+  {
+    padding: '14px',
+    ':hover':
+    {
+      backgroundColor: '#5b616b',
```
Comment (mdbenjam) [inline client/app/reader/DropdownFilter.jsx]: Leaving as a magic number until we have these colors in CFT

File: app/assets/stylesheets/reader/_document_list.scss [hunk 2]
```diff
@@ -221,63 +221,6 @@ $tag-backgrond-color: #d6d7d9;
         border-bottom-color: $color-gray;
       }
     }
-
-    .cf-document-tag-picker {
```
Comment (nickheiner-usds) [inline app/assets/stylesheets/reader/_document_list.scss]: image

File: client/app/reader/DocCategoryPicker.jsx [hunk 3]
```diff
@@ -31,15 +43,49 @@ CategorySelector.propTypes = {
...
 
-const DocCategoryPicker = ({ categoryToggleStates, handleCategoryToggle, allowReadOnly }) => {
-  return <ul className="cf-document-filter-picker cf-document-category-picker">
+const docCategoryPickerStyle = css(
+  {
+    listStyleType: 'none',
+    paddingLeft: 0,
+    paddingBottom: 0
+  },
+  select('& li',
```
Comment (nickheiner-usds) [inline client/app/reader/DocCategoryPicker.jsx]: What is the point of `select`? The glamor docs suggest that it works with or without `select` 🙃 

```js
import { css } from 'glamor'

let box = css({
  display: 'block',
  '& .bold': { fontWeight: 'bold' },
  '& .one': { color: 'blue' },
  ':hover .two': { color: 'red' }
})

// or 
import { css, select as $ } from 'glamor'
let box = css(
  { display: 'block' },
  $('& .bold', { fontWeight: 'bold' }),
  $('& .one', { color: 'blue' }),
  $(':hover .two', { color: 'red' }),  
)
```

File: client/app/reader/DropdownFilter.jsx [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+      color: '#ffffff'
```
Comment (nickheiner-usds) [inline client/app/reader/DropdownFilter.jsx]: We do have `WHITE` in c-f-t.

**Pull request merged**